Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    apivore (1.0.0)
+    apivore (1.1.0)
```

Comment:
It is not recommended to check in the `Gemfile.lock` Be good to get this removed from version control. See  http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/